docs/source/user-guide/guide_file_fetchers.rst

```diff
@@ -0,0 +1,57 @@
+Implementing a custom file fetcher
+----------------------------------
+
+DKAN uses a library called getdkan/file-fetcher. This library allows developers to extend the file transfer functionality for their specialized needs.
+
+This library is used to download a resource, such as a CSV file, so that it can be loaded into the database and presented through the UI and API. This process is called 'localization,' because the source resource is copied to the local file system.
```

Comment:
```suggestion
This library is used to download a resource, such as a CSV file, so that it can be loaded into the database and presented through the UI and API. This process is called "localization," because the source resource is copied to the local file system. Usually, this downloaded copy is temporary and is eventually removed.
```